backend/clubs/models.py

```diff
@@ -1545,6 +1551,12 @@ def __str__(self):
             self.application_end_time,
         )
 
+    @property
+    def season(self):
+        semester = "Fall" if 8 <= self.application_start_time.month <= 11 else "Spring"
```

Comment:
I think this is probably fine for now but I had an idea a little while ago for a `ApplicationCycle` model which would look roughly like:
```
class ApplicationCycle(models.Model):
   start_time=...
   end_time=...
   results_release_time=...
   name=...
   semester=...
```
and then we could do a many-to-one relationship with `ClubApplication`. This would make it a lot easier for us to manually query only the particular applications in any given cycle. It would also make it easier to standardize the `start_time`, `end_time`, and `results_release_time` across all the clubs in a Wharton Council application cycle.

While this is a general model, I'm guessing we'd only use it to group WC application cycles for now.

This `season` function would then become something like
```
@property
def season(self):
   return self.cycle.semester
```

Reply:
Yeah this is a good idea. I'll leave this in a backlog since it'll involve co-ordinating `start_time`, `end_time`, and whatnot with WC.